File: app/src/main/java/com/demo/demoapplication/ui/fragments/gallery/GalleryViewModel.java
package com.demo.demoapplication.ui.fragments.gallery;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

import com.demo.demoapplication.R;

public class GalleryViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public GalleryViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("Hello world 1");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/demo/demoapplication/ui/fragments/gallery/GalleryViewModel.kt
package com.demo.demoapplication.ui.fragments.gallery

import androidx.lifecycle.ViewModel
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.LiveData

class GalleryViewModel : ViewModel() {
    private val mText: MutableLiveData<String>
    val text: LiveData<String>
        get() = mText

    init {
        mText = MutableLiveData()
        mText.value = "Hello world 1"
    }
}